project/src/index.js

```diff
@@ -0,0 +1,34 @@
+import {
+  renderTasks, addItem, removeItem, toggleItem, clearCompleted,
+} from './todo.js';
+
+const addItemButton = document.getElementById('addItemButton');
+const taskList = document.getElementById('taskList');
+const clearCompletedButton = document.getElementById('clearCompletedButton');
+
+addItemButton.addEventListener('click', handleAddItem);
+taskList.addEventListener('click', handleTaskClick);
+clearCompletedButton.addEventListener('click', handleClearCompleted);
+
+function handleAddItem() {
+  addItem();
+}
+
+function handleTaskClick(event) {
+  const { target } = event;
+  if (target.classList.contains('remove-item')) {
+    const listItem = target.closest('li');
+    const index = Array.from(taskList.children).indexOf(listItem);
+    removeItem(index);
+  } else {
+    const listItem = target.closest('li');
+    const index = Array.from(taskList.children).indexOf(listItem);
+    toggleItem(index);
+  }
+}
+
+function handleClearCompleted() {
+  clearCompleted();
+}
+
+renderTasks();
```

Comment:
- [ ] Great job on implementing the `add` and `remove` functionality of your project👏. However, You seem to have not understood the project require for this milestone. Please note that for this milestone of your project, you're required setup `webpack` and also create an `array` of object container **hard-coded** data, and with the help of `forEach()` or `map()` you iterate over the **hard-coded** data and display it to the UI based on their sorted index. You're not required to implement the `add` and remove functionality in this milestone so kindly consider removing those functionalities. See below for more details on project requirements: 
<img width="1074" alt="Screenshot 2023-07-11 at 23 05 36" src="https://github.com/lerfast/To-do-list-webpack/assets/107192076/cef19be1-21f0-4207-a7ea-f8b89a3c066f">

Reply:
Hi, I organized the files and folders in my project like this:
![image](https://github.com/lerfast/To-do-list-webpack/assets/111823606/67241ed0-6cb9-4b3a-bbda-cce646f5534d)
Also I did the changes in my code that you require, I left a remove icon as i see in the video.

![image](https://github.com/lerfast/To-do-list-webpack/assets/111823606/bfe60b0c-aade-4604-b904-3163c2d4bd3e)

I hope that this is ok, I did my best, thanks